tests/fields/fields.go

```diff
@@ -17,13 +17,13 @@
 package main
 
 import (
+	"code.google.com/p/goauth2/oauth"
```

Comment:
I think `goimports` would move this to the middle like this:

```
    "encoding/json"
    "flag"
    "fmt"
    "os"
    "reflect"
    "strings"

    "code.google.com/p/goauth2/oauth"

    "github.com/google/go-github/github"
```

but this commit was made with `gofmt -s -w .`